pkg/distro/distro_test.go

```diff
@@ -46,6 +46,10 @@ func TestImageType_PackageSetsChains(t *testing.T) {
 							URL: "https://example.com", // required by some image types
 						},
 					}
+					if imageType.Name() == "coreos-qcow2" {
+						options.OSTree.Container = "https://registry.example.com/image"
```

Comment:
is this supposed to be a container pullspec? those don't often include `https://` do they?

Reply:
It could be any string since the test doesn't really do anything with it, but you're right.  Might as well make it look like a real spec.